src/radiation/monte_carlo.cpp

```diff
@@ -280,13 +280,19 @@ TaskStatus MonteCarloSourceParticles(MeshBlock *pmb, MeshBlockData<Real> *rc,
             // Sample energy and set weight
             Real nu;
             int counter = 0;
+            Real prob;
             do {
               nu = exp(rng_gen.drand() * (lnu_max - lnu_min) + lnu_min);
```

Comment:
For efficiency, sample `lnu` here---don't take the exponent. Then you can skip the `lnu = log(nu)` line below. Then after the rejection sample in the while loop, compute `nu = exp(lnu)`

Reply:
Yeah good point -- fixed.